Resources/public/css/layout.css

```diff
@@ -29,6 +29,7 @@
     opacity: 1;
     min-height: 100%;
     overflow: auto;
+    pointer-events: all;
```

Comment:
Well, where is it placed doesn't really matter. The `is-app-open` CSS class is added as soon as the Loading application screen disappears.

We can keep the `.is-app-loading` selector alone without `.is-app-open` at the beggining of the selector.
I did it this way to show, that `.is-app-loading` class is added when the app is opened but switching active views.

To sumarize, I'd keep it here.